qiskit_ibm_provider/ibm_backend_service.py

```diff
@@ -225,6 +226,7 @@ def jobs(
             descending: If ``True``, return the jobs in descending order of the job
                 creation date (i.e. newest first) until the limit is reached.
             instance: The provider in the hub/group/project format.
+            legacy: Filter to only retrieve jobs run from the deprecated `qiskit-ibmq-provider`.
```

Comment:
```suggestion
            legacy: If ``True``, only retrieve jobs run from the deprecated ``qiskit-ibmq-provider``. Otherwise retrieve only jobs run from ``qiskit-ibm-runtime``.
```
Make it clearer `legacy=False` doesn't retrieve jobs from both.